swarm

```diff
@@ -347,6 +351,14 @@ haproxy() { # Owner: freignat91
     appcelerator/haproxy:latest
 }
 
+nats-streaming() { # Owner: bquenin
```

Comment:
It's a legal name in bash, but not recommended for shell variables (should just be letters, digits, and underscores) and definitely not legal in `sh`.